### ISSUE TITLE
New Feature: screenshot 

### ISSUE THREAD

**Issue by Laisky** [RocketChat/Rocket.Chat#3074]
Your Rocket.Chat version: v0.28

It could be very useful if user can take a screenshots in chats(channels or private groups).

for example, add a screenshot button near the "file upload button". after click the screenshot button, people can drag and capture the region they choosed.

**Comment by Laisky** [comment]
click submit by accident...

**Comment by sadeva** [comment]
Hi @Laisky, I don't think if we need this feature. You can do that using `PrintScreen` utility on your OS, or use a browser extension.

**Comment by marceloschmidt** [comment]
I agree with @sadeva 
Why would you need such a thing, @Laisky ?

**Comment by twhiting** [comment]
@Laisky if you on windows use the *snipping tool* its exactly what your looking for

**Comment by Laisky** [comment]
I thought there is no simple way to capture a region of the screen on Windows, thanks for @twhiting , now I know `snipping tool`, I will append this to my company's IM tutorial.

The situation is:

I want to promote rocket.chat to my company, but many people ask me where is the screenshot button. because many other IMs have the screenshot feature, the people in my company used to capture the screen by IM toolbar.

If rocket.chat do not support screenshot feature, these people forced to change their habits. So I think it's better to ask you first.

If you really think there is no need to add this feature, you can close this issue. then I will tell my users to use rocket.chat & snipping tool to replace the original IM.

however, thank you for your work!

**Comment by marceloschmidt** [comment]
Let's close this one and keep track of it on #3047. This is a duplicate.

**Comment by takavfx** [comment]
I want this one too as a function installed into Rocket.Chat or Rocket.Chat.Electron.
I've already posted my idea here: https://github.com/RocketChat/Rocket.Chat.Electron/issues/334

Actually I found this( https://www.npmjs.com/package/electron-screencapture ) npm-package for electron. I'm not sure how this works but it can be better some kind of this thing is installed into such as  Rocket.Chat.Electron.
